README.md

```diff
@@ -609,45 +672,72 @@ cases?
+The infrastructure is specified as a separate
+[document](https://wicg.github.io/client-hints-infrastructure), and integration with Fetch and HTML
+happens there.
 
 ## What's with the `Sec-CH-` prefix?
 
 Based on some discussion in [w3ctag/design-reviews#320](https://github.com/w3ctag/design-reviews/issues/320#issuecomment-435874298),
 it seems reasonable to forbid access to these headers from JavaScript, and demarcate them as
-browser-controlled client hints so they can be documented and included in requests without triggering
-CORS preflights. A `Sec-CH-` prefix seems like a viable approach.
+browser-controlled client hints so they can be documented and included in requests without
+triggering CORS preflights. A `Sec-CH-` prefix seems like a viable approach.
 
 ## How does `Sec-CH-UA-Mobile` define "mobile"?
 
 This is a tough question. The motivation for the header is that a majority of user-agent header
 sniffing is used by the server to decide if a "desktop" or "mobile" UX should be served. This is
 currently implicitly defined in most modern browsers because they have two distinct UIs, a
-"desktop" version (i.e. Windows, Mac OS, etc.) and a "mobile" version (i.e. Android, iOS). In general,
-most browsers will also explicitly send "Mobile" in user-agent strings on "mobile" platforms. As
-such, servers will usually use the platform or presence of this "mobile" identifier. It's also worth
-pointing out that most modern browsers also have an explicit "request desktop site" UI element in their
-mobile versions which should be honored. In a more general sense, though, a "mobile" experience could be
-seen as a UX designed with smaller screens and touch-based interface in mind.
+"desktop" version (i.e. Windows, Mac OS, etc.) and a "mobile" version (i.e. Android, iOS). In
+general, most browsers will also explicitly send "Mobile" in user-agent strings on "mobile"
+platforms. As such, servers will usually use the platform or presence of this "mobile" identifier.
+It's also worth pointing out that most modern browsers also have an explicit "request desktop site"
+UI element in their mobile versions which should be honored. In a more general sense, though, a
+"mobile" experience could be seen as a UX designed with smaller screens and touch-based interface
+in mind.
+
+## Aren’t we duplicating a lot of information already in the `User-Agent` header?
+
+It’s true that for some period of time there will be redundancy between the `User-Agent` header and
+the hints for platform, browser name and version, and mobileness. Over time, it may be possible to
+further freeze the User-Agent header to a single static string as the web platform adopts to Client
+Hints.
+
+## Aren’t you adding a lot of new headers? Isn’t that going to bloat requests?
+
+It’s true that we are adding up to 8 new headers per request. That’s quite a lot. But we don’t
```

Comment:
Maybe consider leaving the number out so we don't have to remember to change it if a new header gets added

Reply:
Yep, great feedback. Let's just say "multiple new headers".